pkg/builtins/range_check.go

```diff
@@ -36,14 +36,18 @@ type RangeCheckBuiltinRunner struct {
 	ratio                 uint
 	instancesPerComponent uint
 	StopPtr               *uint
+	nParts                uint
+	Bound                 lambdaworks.Felt
```

Comment:
`nParts` & `Bound` are both constant across all layouts, so lets use the constants instead of adding fields to the builtin runner